src/main/java/com/kuljava/swiatwsi/world/Point.java

```diff
@@ -0,0 +1,26 @@
+package com.kuljava.swiatwsi.world;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Getter
+@Table
```

Comment:
According to JEE docs, if no @ Table annotation is present, Hibernate will take default values. Sooo... If you are not overriding anything here, this annotation is useless.